A few clarifications on multiplying by adjoints (#847)
QuantEcon/lecture-source-jl: source/rst/getting_started_julia/fundamental_types.rst

.. code-block:: julia
    :class: skip-test

    ones(2) * ones(2)  # does not conform, expect error

Instead, you could take the transpose to form a row vector

.. code-block:: julia

    ones(2)' * ones(2)

> Alternatively, you can use an inner product in this setting use ``dot()`` or the unicode ``\cdot<TAB>``

Reviewer: ```suggestion
Alternatively, for inner product in this setting use ``dot()`` or the unicode ``\cdot<TAB>``
```

.. code-block:: julia

    @show dot(ones(2), ones(2))
    @show ones(2) ⋅ ones(2);

> Matrix multiplication using one dimensional vectors similarly follow from treating them as

Reviewer: ```suggestion
Matrix multiplication using one dimensional vectors similarly follows from treating them as
```
